src/ui/molecules/checkbox/index.tsx

```diff
@@ -1,49 +1,36 @@
 import * as React from 'react';
 import styled, { StyledComponent } from 'styled-components';
+import { CheckIcon, UnCheckIcon } from 'icons';
 import { Variant } from 'lib/types';
 
 interface CheckboxProps {
   className?: string;
+  disabled?: boolean;
   id: string;
   isChecked: boolean;
-  label?: React.ReactNode;
   onChange: React.ChangeEventHandler<HTMLInputElement>;
+  text?: string;
 }
 
 const CheckboxBase: React.FC<CheckboxProps & Variant> = ({
   className,
+  disabled,
   id,
   isChecked,
-  label,
   onChange,
+  text,
   variant = 'default',
 }) => (
-  <CheckboxWrapper htmlFor={id} className={className} data-variant={variant}>
-    <input type="checkbox" onChange={onChange} id={id} checked={isChecked} />
-    {label && <span>{label}</span>}
-  </CheckboxWrapper>
+  <label htmlFor={id} className={className} data-variant={variant}>
+    <span data-block="container" data-disabled={disabled}>
+      <input type="checkbox" id={id} checked={isChecked} onChange={onChange} />
+      <span data-block="checkmark-unchecked">{<UnCheckIcon />}</span>
+      <span data-block="checkmark-checked">{<CheckIcon />}</span>
```

Comment:
думаю эти два компонента можно пометить слегка удобнее:
```suggestion
      <span data-checkmark="unchecked">{<UnCheckIcon />}</span>
      <span data-checkmark="checked">{<CheckIcon />}</span>
```